docs/configuration_file.md

```diff
@@ -155,6 +155,25 @@ thread_count = <em>&lt;thread_count&gt;</em>
 
 Note: the `thread_count` parameter is optional and denotes the number of parallel threads used to by the plugin to connect to the endpoint. The higher this number, the faster the execution, however if the number is too high the server may interrupt the connection with a rate-limit error.
 
+### Kraken Section (REST)
+This plugin is REST-based and requires setting up API Keys in your Kraken account settings (click on the API link).
```

Comment:
The "API link" comes from the Coinbase plugin: just wanted to make sure that Kraken also has a link called "API" and this is not a copy/paste error.